style.css

```diff
@@ -64,4 +64,147 @@ body
     {
         display: none;
     }
+}
+
+/* Second section top cover */
+
+.top-cover
+{
+    position: relative;
+    height: 800px;
+    background-image: url(/images/cover-background-black-tower.jpg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    max-width: 100%;
+}
+
+.navbar
+{
+    display: flex;
+    flex-direction: row;
+    justify-content: end;
+    align-items: center;
+    width: 100%;
+    height: 4.25rem;
+    margin: 0 auto;
+    padding-top: 50px;
+    padding-bottom: 50px;
+}
+
+.navbar .logo
+{
+    margin-right: 250px;
```

Comment:
if you give flex properly no need for this. the revist the styles in your continer class. it will solve all your issue